### ISSUE TITLE
ERROR Error: missing field `tag_name` at line 1 column 280

### ISSUE THREAD

**Issue by emmahyde** [MordechaiHadad/bob#26]
```bash
❯ bob install stable
May 07 17:47:09.380 ERROR Error: missing field `tag_name` at line 1 column 280
```
Sorry, am I misunderstanding the syntax here?

**Comment by MordechaiHadad** [comment]
Hello! This is quite an interesting issue that i have never seen before.
Could you try perhaps downloading other versions and see if you get the same error?
It seems that when bob request the stable version github api's doesnt provide the tag_name field in the sent JSON

**Comment by MordechaiHadad** [comment]
closing due to inactivity, feel free to open if the problem persists

**Comment by kristijanhusak** [comment]
This is happening when there is no proper tag on github. Currently I'm trying to install latest nightly, but it's missing https://github.com/neovim/neovim/releases

**Comment by mosheavni** [comment]
Happened to me as well when I uninstalled nightly, for some reason bob couldn't delete all of the binary files so the directory was not deleted, and there was no `bob.json` file so it probably thought that nightly is still installed.
You should add a verification that if `bob.json` doesn't exist, or is empty, delete the entire nightly (or which ever version) directory and let the install process create it again.

**Comment by MordechaiHadad** [comment]
@kristijanhusak @mosheavni hello yeah i recently fixed it(I hope for everyone) with commit f1585abada425874546c0145224f009d17704d3a
I will reopen this issue until it is fully resolved
